Tests/docker_native_image_config.json

```diff
@@ -194,6 +194,16 @@
                 "native:dev",
                 "native:candidate"
             ]
+        },
+        {
+            "id":"CrowdstrikeFalcon",
+            "reason":"urllib",
```

Comment:
@tkatzir if you skip it, please open an issue about it with the `NativeImage` component